.ci/jenkins/Jenkinsfile.prod.nightly

```diff
@@ -14,7 +14,7 @@ pipeline {
         string(description: 'The UMB message version', name: 'UMB_VERSION', defaultValue: 'main')
         string(description: 'The product version, if not provided the optaplanner main branch one will be used', name: 'PRODUCT_VERSION')
         string(description: 'The drools product version, if not provided the drools main branch one will be used', name: 'DROOLS_PRODUCT_VERSION')
-        string(description: 'The config repository branch', name: 'CONFIG_BRANCH', defaultValue: 'main')
+        string(description: 'The config repository branch', name: 'CONFIG_BRANCH', defaultValue: 'master')
```

Comment:
so this is the eng-jenkins JCasC configuration branch, right? just to confirm it.

Reply:
yeah exactly, refer to this https://gitlab.cee.redhat.com/middleware/build-configurations/-/tree/master/rhbop

Reply:
it refers to the PNC build configuration repo, which has `master` as main branch